.changeset/nasty-geckos-hide.md

```diff
@@ -0,0 +1,7 @@
+---
+"@aws-amplify/ui-react-core-notifications": patch
+"@aws-amplify/ui-react-core": patch
+"@aws-amplify/ui-react": patch
+---
+
+Add deprecation messages for InAppMessagingDisplay, InAppMessagingProvider and useInAppMessaging.
```

Comment:
Can we add more details here? Some instructions on how customers can migrate would be great here.

Reply:
Updated changeset to include more details.